### PR TITLE
For you to Following

### PR DESCRIPTION
**Pull request by msamgan**
#453

image

### CODE REVIEW

**Review by MrPunyapal — CHANGES_REQUESTED** [review]

File: app/Queries/Feeds/QuestionsFollowingFeed.php [hunk 1]
```diff
@@ -8,10 +8,10 @@
 use App\Models\User;
 use Illuminate\Database\Eloquent\Builder;
 
-final readonly class QuestionsForYouFeed
+final readonly class QuestionsFollowingFeed
```
[the remark this answers is not in the document]
Comment (msamgan) [reply app/Queries/Feeds/QuestionsFollowingFeed.php:11]: I hear you, but the logic for follower and following is not touched at all. That makes the need to change in query, not required.

Its just the label and files names which will be impacted. At DB level there is no change hence no update in Query.

what am I missing ?

[the remark this answers is not in the document]
Comment (MrPunyapal) [reply app/Queries/Feeds/QuestionsFollowingFeed.php:11]: it should show what exactly the following means.

[the remark this answers is not in the document]
Comment (msamgan) [reply app/Queries/Feeds/QuestionsFollowingFeed.php:11]: can you please elaborate for better understanding ?

[the remark this answers is not in the document]
Comment (msamgan) [reply app/Queries/Feeds/QuestionsFollowingFeed.php:11]: image


Its exactly what you are saying.  the query is on $question->to only.

[the remark this answers is not in the document]
Comment (ojessecruz) [reply app/Queries/Feeds/QuestionsFollowingFeed.php:11]: @msamgan it's the same logic on Twitter/X. The Following tab, should show just the posts from the users you are currently following.

What @MrPunyapal is saying, is you need to change the entire logic here to make sense to change the Tab's name,

[the remark this answers is not in the document]
Comment (msamgan) [reply app/Queries/Feeds/QuestionsFollowingFeed.php:11]: thanks for the input @ojessecruz . The I got that part, but the current query is doing exactly the same. That is what i did not undertand. What are the changes required.

The for you page right now shows the post from the people you are are following. The last line in the query (image above) shows the same. This is what I am confused about.

[the remark this answers is not in the document]
Comment (MrPunyapal) [reply app/Queries/Feeds/QuestionsFollowingFeed.php:11]: When you see orWhere that means it's not just including following posts it has some extra sutff that we need to remove 🙂

[the remark this answers is not in the document]
Comment (lpheller) [reply app/Queries/Feeds/QuestionsFollowingFeed.php:11]: No, the query loads records that are created by people who’s posts you liked in the past (60 days). 

If I liked one of your posts / questions yesterday but I’m not following you - your posts would still appear in the for you page.

[the remark this answers is not in the document]
Comment (msamgan) [reply app/Queries/Feeds/QuestionsFollowingFeed.php:11]: Ohhh.. This is what you meant. You meant only the post from followings. Right now it also shows the liked and replied posts by the following. Thanks for the explanation. I'll make the update tomorrow.

Thanks @MrPunyapal

[the remark this answers is not in the document]
Comment (MrPunyapal) [reply app/Queries/Feeds/QuestionsFollowingFeed.php:11]: @msamgan are you there?

[the remark this answers is not in the document]
Comment (msamgan) [reply app/Queries/Feeds/QuestionsFollowingFeed.php:11]: yes, was preoccupied in some family matter.
Will give you an updated PR by tomorrow...

**Pull request closed without merging**